parquet/src/file/properties.rs

```diff
@@ -299,7 +299,7 @@ impl WriterProperties {
             .unwrap_or(DEFAULT_DICTIONARY_ENABLED)
     }
 
-    /// Returns `true` if statistics are enabled for a column.
+    /// Returns the type of  statistics written for a column.
```

Comment:
```suggestion
    /// Returns which statistics are written for a column.
```

seems more accurate, as there are two different types of statistics and they are not mutually exclusive